lib/chunks.js

```diff
@@ -1065,3 +1077,34 @@ module.exports.getTemplateQuestionIdsAsync = async (question) => {
 module.exports.getTemplateQuestionIds = util.callbackify(
   module.exports.getTemplateQuestionIdsAsync
 );
+
+/**
+ * Logs the changes to chunks for a given job.
+ *
+ * @param {{ updatedChunks: ChunkMetadata[], deletedChunks: ChunkMetadata[] }} chunkMetadata
+ * @param {import('./server-jobs').Job} job
+ * @returns
+ */
+module.exports.logChunkChangesToJob = ({ updatedChunks, deletedChunks }, job) => {
+  if (updatedChunks.length === 0 && deletedChunks.length === 0) {
+    return;
```

Comment:
Perhaps it would be useful to explicitly record that we did not update/delete chunks, just to be extra clear (see below)?